scripts/docker-start-kafka.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# this script is used when the docker kafka container starts
+# this version is specialized from
+# https://github.com/wurstmeister/kafka-docker/blob/master/start-kafka.sh
+# to work with our setup (esp. with ssl)
+
+set -ev
+
+cp ${KAFKA_HOME}/config/server.properties.in ${KAFKA_HOME}/config/server.properties
+
+if [[ -z "$KAFKA_LOG_DIRS" ]]; then
+    export KAFKA_LOG_DIRS="/kafka/kafka-logs-$HOSTNAME"
+fi
+
+if [[ -n "$KAFKA_HEAP_OPTS" ]]; then
+    sed -r -i "s/(export KAFKA_HEAP_OPTS)=\"(.*)\"/\1=\"$KAFKA_HEAP_OPTS\"/g" $KAFKA_HOME/bin/kafka-server-start.sh
+    unset KAFKA_HEAP_OPTS
+fi
+
+# any env vars starting with KAFKA will be set in server.properties
+#   e.g., KAFKA_LOG_DIRS =>log.dirs
+# if the var is already in server.properties, it will be replaced
+# if it is not already in server.properties, it will be added to the end
+for VAR in `env`
```

Comment:
the env var substitution is not necessary since we are not setting them anywhere, same with the `KAFKA_HEAP_OPTS` from the previous `if` condition